components/ADInputFile.vue

```diff
@@ -12,7 +13,7 @@ const props = defineProps({
     default: 300,
   },
   medias: {
-    type: Array<String>,
+    type: Object as PropType<Array<String> | String>,
```

Comment:
```suggestion
    type: Object as PropType<Array<string> | string>,
```
Sinon ça cause des erreurs à l'utilisation, je pourrais close https://github.com/add-eus/ui-kit/pull/10